README.md

```diff
@@ -14,8 +14,19 @@ Turn this into trading application
 
 - If you are new to React please you go through react documents and we can discuss from there.
 - Clone the repo and go through the code base.
+- Make sure you have node installed on your computer.
 - Go through the issues and find the one you find interesting.
 - Discuss and send the PR.
 - If you need any other help, feel free to ping on crypto channel at discord.
 
+**Installation**
+* `git clone<repository-url>` this repository
+* `cd website-crypto`
+* `npm install`
+
+**Runing/Development**
+* `npm run build`
+* `npm run start`
```

Comment:
`build` is for production build
`start` is to serve the the production build, made by `build`

We use `dev` for development

Can you please make these changes?

Thank you for the pull request! 🥳

Reply:
Please review it once again I made the changes.

Reply:
Please read that again.... We use `dev` for development purposes

Reply:
Sorry for the inconvenience. I think the required changes are made please review.

Reply:
Almost there. Shouldn't have removed `build` and `start`

You can mention these two commands below the dev one. Saying
To build: npm run build
To serve the production build: npm run start

Make sure these are AFTER the dev command, so users are not confused. Thank you so much!

I'd like to invite you to our Discord Server 😉
We have such good projects which are in React, Ember, Vue for frontend and Node, Express in backend. We keep having technical and non-technical discussions there. Here's the link to join:  https://discord.gg/CjwSPnjB

Reply:
Thank you for your guidance. The said changes are made please review.